.mailmap

```diff
@@ -0,0 +1,31 @@
+Andrew Myers <atmyers@lbl.gov>              Andrew Myers <atmyers2@gmail.com>
+Andrew Myers <atmyers@lbl.gov>              atmyers <atmyers2@gmail.com>
+Aurore Blelly <ablelly@lbl.gov>             ablelly <aurore.blelly@ensta-paristech.fr>
+Aurore Blelly <ablelly@lbl.gov>             ablelly <48810880+ablelly@users.noreply.github.com>
+Axel Huebl <axelhuebl@lbl.gov>              Axel Huebl <axel.huebl@plasma.ninja>
+David Grote <dpgrote@lbl.gov>               grote <dpgrote@lbl.gov>
+David Grote <dpgrote@lbl.gov>               Dave <grote1@llnl.gov>
+David Grote <dpgrote@lbl.gov>               Dave Grote <grote1@llnl.gov>
```

Comment:
@dpgrote do you prefer the LLNL or LBL e-mail as primary here (left-hand side)

Reply:
The lbl.gov email address is fine. This looks good. I'll setup my git profile to use that email.

Reply:
Okay, thanks! Please do not forget to approve the PR :)